packages/garbo/src/config.ts

```diff
@@ -229,3 +239,17 @@ You can use multiple options in conjunction, e.g. "garbo nobarf ascend"',
   },
   { positionalArgs: ["turns"] },
 );
+
+type GlobalOptions = typeof globalOptions;
+
+export function isAnyQuick(opts: GlobalOptions): boolean {
+  return opts.quick || opts.quickgear || opts.quickcombat;
+}
+
+export function isQuickGear(opts: GlobalOptions): boolean {
+  return opts.quick || opts.quickgear;
+}
+
+export function isQuickCombat(opts: GlobalOptions): boolean {
+  return opts.quick || opts.quickcombat;
+}
```

Comment:
We're only ever going to have one `globalOptions` object--why pass this as an arg instead of just making these functions reference the thing to begin with?